File: eclipse-workspace/Java05_switch/src/com/test02/Score.java
package com.test02;

public class Score {
	public double getAvg(int kor, int eng , int math) {
		double avg = (kor + eng + math) / 3.0;
		return avg;
	}
	
	public String getGrade(double avg) {
		String grade;
		
		//100/10은 10 과 같이 점수를 10으로 나누어 case로 구분
		switch((int)avg/10) {
		case 10 :
		case 9 :
			grade = "A";
			break;
		case 8 : 
		case 7 :
			grade = "B";
			break;
		default : 
			grade = "F";
		}
		 
		return grade;
	}
}

File: workspace_WebServer/.metadata/.plugins/org.eclipse.wst.server.core/tmp0/wtpwebapps/Ajax02_Practice/score.js
function getParameterValues() {
	var name = "name="+$("#name").val();
	var kor = "kor="+$("#kor").val();
	var eng = "eng="+$("#eng").val();
	var math = "math="+$("#math").val();
	
	return "?"+name+"&"+kor+"&"+eng+"&"+math0
}

function load() {
	var url = "CalScore"+getparameterValues();
	
	httpRequest = new XMLHttpRequest();
	httpRequest.onreadystatechange=callback;
	httpRequest.open("GET" , url , true);
	httpRequest.send();
}
function callback() {
	alert("readyState : " + httpRequest.readyState);
	
	if(httpRequest.readyState == 4 ) {
		alert("status : " + httpRequest.status);
		
		if(httpRequest.status == 200) {
			var obj = JSON.parse(httpRequest.responseText);
			$("#result").html(obj.name+ " 의 총점은 " + obj.sum + " 이고 , 평균은 " + obj.avg + " 입니다.");
		} else {
			alert("데이터를 계산 할 수 없습니다.");
		}
	}
}
$(function () {
	$("#process").submit(function () {
		load();
	});
});